dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -58,12 +61,40 @@ inline std::pair<size_t, size_t> minmax(const IColumn & column, const ColumnVect
 
     return {batch_min_idx, batch_max_idx};
 }
+
+// minmax calculation for the nullable column
+inline std::pair<size_t, size_t> minmax(const IColumn & column, const ColumnVector<UInt8> * del_mark, const PaddedPODArray<UInt8> & null_mark, size_t offset, size_t limit)
```

Comment:
Better to add comment that this function actually bypasses the NULL (does not treat NULL as what its semantic is supposed to be) and then calculate a MinMax only for not NULL values.

It is definitely not a function calculating MinMax for a nullable column.